plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -335,7 +336,11 @@ public void launch(ILaunchConfiguration configuration, String mode, final ILaunc
 
     LocalAppEngineServerBehaviour serverBehaviour = (LocalAppEngineServerBehaviour) server
         .loadAdapter(LocalAppEngineServerBehaviour.class, null);
-
+    
+    IVMInstall vmInstall = getVMInstall(configuration);
```

Comment:
Might be worth adding a note here that we don't use any other information from the `IVMInstall`, such as VM args, boot libraries, etc.
https://github.com/GoogleCloudPlatform/appengine-plugins-core/issues/353

LGTM.